helpfuls/type_hints.py

```
@@ -0,0 +1,34 @@
+text: str = "Value"
+age: int = 55
+temp: float = 22
+
+number: int | float = 12
+
+optional: str | None
+
+optional = "check it"
+
+
+def root(num: int, float, exp: float | None) -> float:
```

Ruff (0.13.1)

Unused function argument: `float`

(ARG001)